src/main/java/picard/sam/SamAlignmentMerger.java

```diff
@@ -194,6 +186,27 @@ public void mergeAlignment(final File referenceFasta) {
         }
     }
 
+    @Override
+    protected SAMSequenceDictionary getDictionaryForMergedBam() {
+        SAMSequenceDictionary finalDict;
+        if (alignedSamFile != null && !alignedSamFile.isEmpty()) {
+            finalDict = SAMSequenceDictionaryExtractor.extractDictionary(alignedSamFile.get(0));
+            alignedSamFile.stream()
+                    .map(SAMSequenceDictionaryExtractor::extractDictionary)
+                    .forEach(finalDict::assertSameDictionary);
+        } else {
+            final SeparateEndAlignmentIterator mergingIterator = new SeparateEndAlignmentIterator(this.read1AlignedSamFile, this.read2AlignedSamFile, referenceFasta);
+            finalDict = mergingIterator.getHeader().getSequenceDictionary();
+        }
+
+        SAMSequenceDictionary referenceDict = SAMSequenceDictionaryExtractor.extractDictionary(referenceFasta);
+        if (referenceDict == null) {
+            throw new PicardException("No sequence dictionary found for " + referenceFasta.getAbsolutePath() +
+                    ".  Use CreateSequenceDictionary.jar to create a sequence dictionary.");
```

Comment:
This message is horribly out of date ;)  `Use Picard's CreateSequenceDictionary`